text/0070-x-track-kusamanetwork.md

```diff
@@ -0,0 +1,160 @@
+make posts once a referendum is enacted. As stated above, we really only need a system.remark (or batch of remarks) to indicate the contents of a proposed X post.
+The most straight-forward way to do this is to require remarks to adhere to X's requirements for making [posts via their API](https://developer.twitter.com/en/docs/twitter-api/tweets/manage-tweets/api-reference/post-tweets).
+
+For example, if I wanted to propose a post that contained the text "Hello World!" I would propose a referendum in the X post track that contains the following call data:
+`0x0000607b2274657874223a202248656c6c6f20576f726c6421227d` (i.e. `system.remark('{"text": "Hello World!"}')`).
+
+At first, we could support text posts only to prove the concept. Later on we could expand this spec to add support for media, likes, retweets, replies, polls, and
+whatever other X features we want.
+
+### Phase 3 - Release, Tooling, & Documentation
+
+Once we agree on track configurations and specs for referenda in this track, the Fellowship can move forward with merging these changes into Kusama's runtime and
+include them in its next release. We could also move forward with developing the necessary tools that would listen for enacted referenda to post automatically on X.
+This would require coordination with whoever controls the X account; they would either need to run the tools themselves or add a third party as an authorized user to
+run the tools to make posts on the account's behalf. This is a bottleneck for decentralization, but as long as the tools are run by the X account manager or by a trusted third party
+it should be fine. I'm open to more decentralized solutions, but those always come at a cost of complexity.
+
+For the tools themselves, we could open a bounty on Kusama for developers/teams to bid on. We could also just ask the community to step up with a Treasury proposal
+to have anyone fund the build. Or, the Fellowship could make the release of these changes contingent on their endorsement of developers/teams to build these tools. Lots of options!
+For the record, me and my team could develop all the necessary tools, but all because I'm proposing these changes doesn't entitle me to funds to build the tools needed
+to implement them. Here's what would be needed:
+
+- a listener tool that would listen for enacted referenda in this track, verify the format of the remark(s), and submit to X's API with authenticating credentials
+- a UI to allow layman users to propose referenda on this track
+
+After everything is complete, we can update the Kusama wiki to include documentation on the X post specifications and include links to the tools/UI.
+
+## Drawbacks
+
+The main drawback to this change is that it requires a lot of off-chain coordination. It's easy enough to include the track on Kusama but it's a totally different
```

Comment:
While reading this, why not post onto subsocial? I assume that this should probably work using a XCM message? We could then just have some kind of mirror from subsocial to twitter if wanted.

Reply:
This is a very clever solution! I wonder if @siman could provide insight here or recommend a good way to do this.